extension/src/popup/components/account/AccountHeader.tsx

```diff
@@ -0,0 +1,198 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+
+import { COLOR_PALETTE, ROUNDED_CORNERS } from "popup/constants/styles";
+import { ROUTES } from "popup/constants/routes";
+
+import { AccountListIdenticon } from "popup/components/identicons/AccountListIdenticon";
+import { settingsNetworkDetailsSelector } from "popup/ducks/settings";
+
+import { Account } from "@shared/api/types";
+import CreateNewIcon from "popup/assets/create-new.svg";
+import ImportNewIcon from "popup/assets/import-new.svg";
+
+const AccountHeaderSection = styled.div`
```

Comment:
can we convert these styled components into CSS? It can be done in a followup PR, if you prefer

Reply:
ya  the account header styling still needs some work, especially the dropdown part, so I was gonna tackle that + moving to scss in a followup PR if that's alright

Reply:
Yup! Totally fine